scanners/a11y.py

```diff
@@ -6,8 +6,7 @@
 from scanners import utils
 
 
-workers = 25
-PA11Y_STANDARD = 'WCAG2AA'
+workers = 1
```

Comment:
Hah. Totally inherited that change.

@adelevie, is there a reason not to parallelize this?